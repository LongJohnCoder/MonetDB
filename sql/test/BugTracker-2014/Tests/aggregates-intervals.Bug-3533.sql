START TRANSACTION;
CREATE TABLE INTERVAL_TBL (f1 interval second);
INSERT INTO INTERVAL_TBL (f1) VALUES (1), (2), (3), (4);
SELECT count(f1), cast(sum(f1) as bigint), avg(f1), median(f1), max(f1), min(f1) FROM INTERVAL_TBL;

CREATE TABLE INTERVAL_TBL2 (f1 interval month);
INSERT INTO INTERVAL_TBL2 (f1) VALUES (1), (2), (3), (4);
SELECT count(f1), cast(sum(f1) as bigint), avg(f1), median(f1), max(f1), min(f1) FROM INTERVAL_TBL2;
ROLLBACK;
