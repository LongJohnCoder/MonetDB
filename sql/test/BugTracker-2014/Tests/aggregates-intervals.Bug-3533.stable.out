stdout of test 'aggregates-intervals.Bug-3533` in directory 'sql/test/BugTracker-2014` itself:


# 16:49:00 >  
# 16:49:00 >  "mserver5" "--debug=10" "--set" "gdk_nr_threads=0" "--set" "mapi_open=true" "--set" "mapi_port=34676" "--set" "mapi_usock=/var/tmp/mtest-284678/.s.monetdb.34676" "--forcemito" "--dbpath=/home/ferreira/repositories/MonetDB-Nov2019/BUILD/var/MonetDB/mTests_sql_test_BugTracker-2014" "--set" "embedded_c=true"
# 16:49:00 >  

# MonetDB 5 server v11.35.0 (hg id: 957d7b1a190c)
# This is an unreleased version
# Serving database 'mTests_sql_test_BugTracker-2014', using 8 threads
# Compiled for x86_64-pc-linux-gnu/64bit with 128bit integers
# Found 15.504 GiB available main-memory.
# Copyright (c) 1993 - July 2008 CWI.
# Copyright (c) August 2008 - 2019 MonetDB B.V., all rights reserved
# Visit https://www.monetdb.org/ for further information
# Listening for connection requests on mapi:monetdb://localhost.localdomain:34676/
# Listening for UNIX domain connection requests on mapi:monetdb:///var/tmp/mtest-284678/.s.monetdb.34676
# MonetDB/GIS module loaded
# SQL catalog created, loading sql scripts once
# loading sql script: 09_like.sql
# loading sql script: 10_math.sql
# loading sql script: 12_url.sql
# loading sql script: 13_date.sql
# loading sql script: 14_inet.sql
# loading sql script: 15_querylog.sql
# loading sql script: 16_tracelog.sql
# loading sql script: 17_temporal.sql
# loading sql script: 18_index.sql
# loading sql script: 20_vacuum.sql
# loading sql script: 21_dependency_views.sql
# loading sql script: 22_clients.sql
# loading sql script: 23_skyserver.sql
# loading sql script: 25_debug.sql
# loading sql script: 26_sysmon.sql
# loading sql script: 27_rejects.sql
# loading sql script: 39_analytics.sql
# loading sql script: 39_analytics_hge.sql
# loading sql script: 40_geom.sql
# loading sql script: 40_json.sql
# loading sql script: 40_json_hge.sql
# loading sql script: 41_md5sum.sql
# loading sql script: 45_uuid.sql
# loading sql script: 46_profiler.sql
# loading sql script: 51_sys_schema_extension.sql
# loading sql script: 60_wlcr.sql
# loading sql script: 61_wlcr.sql
# loading sql script: 72_fits.sql
# loading sql script: 74_netcdf.sql
# loading sql script: 75_lidar.sql
# loading sql script: 75_shp.sql
# loading sql script: 75_storagemodel.sql
# loading sql script: 80_statistics.sql
# loading sql script: 80_udf.sql
# loading sql script: 80_udf_hge.sql
# loading sql script: 85_bam.sql
# loading sql script: 90_generator.sql
# loading sql script: 90_generator_hge.sql
# loading sql script: 99_system.sql
# MonetDB/SQL module loaded

# 16:49:00 >  
# 16:49:00 >  "mclient" "-lsql" "-ftest" "-tnone" "-Eutf-8" "-i" "-e" "--host=/var/tmp/mtest-284678" "--port=34676"
# 16:49:00 >  

#START TRANSACTION;
#CREATE TABLE INTERVAL_TBL (f1 interval second);
#INSERT INTO INTERVAL_TBL (f1) VALUES (1), (2), (3), (4);
[ 4	]
#SELECT count(f1), cast(sum(f1) as bigint), avg(f1), median(f1), max(f1), min(f1) FROM INTERVAL_TBL;
% sys.%1,	sys.%3,	sys.%4,	sys.%5,	sys.%6,	sys.%7 # table_name
% %1,	%3,	%4,	%5,	%6,	%7 # name
% bigint,	bigint,	double,	sec_interval,	sec_interval,	sec_interval # type
% 1,	5,	24,	5,	5,	5 # length
[ 4,	10000,	2.5,	2.000,	4.000,	1.000	]
#CREATE TABLE INTERVAL_TBL2 (f1 interval month);
#INSERT INTO INTERVAL_TBL2 (f1) VALUES (1), (2), (3), (4);
[ 4	]
#SELECT count(f1), cast(sum(f1) as bigint), avg(f1), median(f1), max(f1), min(f1) FROM INTERVAL_TBL2;
% sys.%1,	sys.%3,	sys.%4,	sys.%5,	sys.%6,	sys.%7 # table_name
% %1,	%3,	%4,	%5,	%6,	%7 # name
% bigint,	bigint,	double,	month_interval,	month_interval,	month_interval # type
% 1,	2,	24,	1,	1,	1 # length
[ 4,	10,	2.5,	2,	4,	1	]
#SELECT median_avg(f1) from INTERVAL_TBL2;
% sys.%1 # table_name
% %1 # name
% double # type
% 24 # length
[ 2.5	]
#DROP TABLE INTERVAL_TBL;
#DROP TABLE INTERVAL_TBL2;

# 16:49:01 >  
# 16:49:01 >  "Done."
# 16:49:01 >  

